src/settings/settings.ts

```diff
@@ -226,8 +226,10 @@ export function getXMLSettings(javaHome: string | undefined, logfile: string, ex
   const isWorkspaceTrusted = (workspace as any).isTrusted;
   if (isWorkspaceTrusted !== undefined && !isWorkspaceTrusted) {
```

Comment:
If you insert a breakpoint here, you'll see there are 2 instances of this settings :

`xml.preferences.downloadExternalRessource`
`xml.downloadExternalResource`

Just make sure whichever namespace is chosen, it is the one the language server is able to parse.

Reply:
I change and fix lemminx with `xml.downloadExternalResources`